### PR TITLE
[using-gatsby-image] Improve default/active nav item contrast, fix SVG size labels

### PR DESCRIPTION
**Pull request by fk**

### CODE REVIEW

File: examples/using-gatsby-image/src/components/navigation.js [hunk 1]
```diff
@@ -17,46 +21,63 @@ const Navigation = () => (
...
       css={{
         fontFamily: options.headerFontFamily.join(`,`),
-        "& a": {
-          color: options.bodyColor,
-          opacity: 0.7,
-          textTransform: `uppercase`,
-          backgroundImage: `none`,
+        margin: 0,
+        listStyle: `none`,
+        "& li": {
```
Comment (KyleAMathews) [inline examples/using-gatsby-image/src/components/navigation.js]: Any reason not to just put these on the component directly?

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 5b36a068046a92bf61cb65daa6db76305057e953

https://app.netlify.com/sites/using-glamor/deploys/59f12d15df9953749d708c8f

**Comment by KyleAMathews** [comment]
I'll merge for now but generally instead of using selectors, just define new components at the top e.g. `NavListItem>` and `<NavLink>`

File: examples/using-gatsby-image/src/components/navigation.js (hunk 1 above)
Comment (fk) [reply examples/using-gatsby-image/src/components/navigation.js]: None but trying to be fast – and being sloppy in consequence.
Will fix!

**Pull request merged**